packages/slate-react/src/plugins/after.js

```diff
@@ -268,7 +268,7 @@ function AfterPlugin() {
     }
```

Comment:
Should this happen here for this `Block` handling logic too @julienp? Could you make that fix too while we're at it to stay consistent?

Reply:
Indeed! I updated the commit.